astropy/visualization/wcsaxes/frame.py

```diff
@@ -137,7 +138,7 @@ def _update_patch_path(self):
     def patch(self):
         self._update_patch_path()
         return PathPatch(self._path, transform=self.parent_axes.transData,
-                         facecolor='white', edgecolor='white')
+                         facecolor=rcParams['axes.facecolor'], edgecolor='white')
```

Comment:
Why not also allow a custom edgecolor from `rcParams`?

Reply:
We draw the patch separately from the spines in WCSAxes - edgecolor is set above on line 102